config/install/core.entity_view_display.rate_plan.rate_plan.teaser.yml

```diff
@@ -0,0 +1,70 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.rate_plan.teaser
+  module:
+    - apigee_m10n
+id: rate_plan.rate_plan.teaser
+targetEntityType: rate_plan
+bundle: rate_plan
+mode: teaser
+content:
+  description:
+    type: string
+    weight: 1
+    region: content
+    label: visually_hidden
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+  displayName:
```

Comment:
I think we should hide this one.  We can do it in the next round, since it's really more specific to the rate plan work and not this plans page.